appveyor.yml

```diff
@@ -0,0 +1,31 @@
+version: '1.0.{build}'
+
+cache:
+  - vendor/bundle
+
+environment:
+  RAILS_ENV: test
+  PGHOST: 127.0.0.1
+  PGUSER: postgres
+  POSTGRES_DB: circuitverse_test
+  PGPASSWORD: Password12!
```

Comment:
I think you can update password here for consistency

Reply:
I don't think it will change the password for the user postgres

Reply:
We can create another database config file `database.appveyor.yml` and use that one instead of `database.example.yml`